src/style.css

```diff
@@ -1 +1,8 @@
 /* styles go here */
+fieldset {
```

Comment:
do not styles tags
checklist
<img width="422" alt="image" src="https://user-images.githubusercontent.com/102411130/228762861-11fbdae0-780f-4542-8f71-3db8f0f486d3.png">